crates/plugin_runtime/src/render_resource_pot/mod.rs

```diff
@@ -14,17 +16,19 @@ use farmfe_core::{
   error::{CompilationError, Result},
   module::{module_graph::ModuleGraph, ModuleId, ModuleSystem},
   parking_lot::Mutex,
-  rayon::prelude::*,
+  rayon::iter::{IntoParallelIterator, ParallelIterator},
   resource::resource_pot::{RenderedModule, ResourcePot},
-  swc_common::{comments::SingleThreadedComments, Mark},
+  swc_common::{comments::SingleThreadedComments, Mark}, // swc_ecma_ast::Function
 };
+use farmfe_plugin_minify::minify_js_module;
```

Comment:
plugins should not be depend on each other, extract minify_js_module to farmfe_toolkit